### PR TITLE
Taxonomy UI

### PR DESCRIPTION
**Pull request by jaypal412**
## Motivation and Context

Fix UI for taxonomy page

## Describe your changes

- Taxonomy Page UI
- Add font icon to add button
- Sticky form buttons
- Stock Location UI
------

## Checklist

- [ ] I have read [CONTRIBUTING.md][contributing].
- [ ] My code follows the [style guidelines][contributing] of this project.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] My code does not generate any (new) [`credo`][credo] and compile-time warnings.
- [ ] I have updated the documentation wherever necessary.
- [ ] I have added tests to cover my changes.

[contributing]: https://github.com/aviabird/snitch/CONTRIBUTING.md
[credo]: https://github.com/rrrene/credo

------
Dear Gatekeeper,

Please make sure that the commits that will be merged or rebased into the base
branch are [formatted according to our standards][commit-style].

> The only additional requirement is that the the PR number must appear in the
> commit title in brackets: `(#XXX)`

[commit-style]: https://github.com/aviabird/snitch/blob/develop/CONTRIBUTING.md#styling

### CODE REVIEW

File: apps/admin_app/lib/admin_app_web/templates/payment_method/new.html.eex [hunk 1]
```diff
@@ -74,3 +74,78 @@
   </div>
 </div>
 
+<div class="list-container">
```
Comment (SagarKarwande) [inline apps/admin_app/lib/admin_app_web/templates/payment_method/new.html.eex]: This template display form for `twice`

File: apps/admin_app/lib/admin_app_web/templates/option_type/index.html.eex [hunk 2]
```diff
@@ -4,8 +4,8 @@
       <h2>Option Types</h2>
     </div>
     <div class="col-2 p-0 float-right text-right">
-      <%= link(" Add a type", to: option_type_path(@conn, :new), 
-        class: "btn btn-primary float-right fas fa-plus")%>
+      <%= link(raw("<i class='fas fa-plus'></i> Add a type"), to: option_type_path(@conn, :new),
```
Comment (SagarKarwande) [inline apps/admin_app/lib/admin_app_web/templates/option_type/index.html.eex]: Change the button text to `Add an Option Type`

File: apps/admin_app/lib/admin_app_web/templates/taxonomy/index.html.eex [hunk 3]
```diff
@@ -1,25 +1,43 @@
...
-  </div>
-  <div class="col-6" align="right">
-    <%= link "Add", to: taxonomy_path(@conn, :new), class: "btn btn-primary" %>
+<div class="list-container">
+  <div class="row m-0 list-header">
+    <div class="col-10 p-0">
+      <h2>Taxonomy</h2>
+    </div>
+    <div class="col-2 p-0 float-right text-right">
+      <%= link(raw("<i class='fas fa-plus'></i> Add a type"), to: taxonomy_path(@conn, :new),
```
Comment (SagarKarwande) [inline apps/admin_app/lib/admin_app_web/templates/taxonomy/index.html.eex]: Change button text to `Add a Taxonomy`

File: apps/admin_app/lib/admin_app_web/templates/taxonomy/taxonomy.html.eex [hunk 4]
```diff
@@ -1,137 +1,65 @@
...
-    <div class="modal-content">
-      <div class="modal-header">
-        <h2 class="modal-title">Add Taxon</h2>
-        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
-          <span aria-hidden="true">&times;</span>
-        </button>
+<div class="list-container">
+  <div class="pt-3 pb-0 pr-3 pl-3 back"> <img src="/images/left-arrow.svg"
+   width="20" height="20" class="d-inline-block align-top" alt=""> 
+   <a href="/prototypes">Taxonomy</a> </div>
```
Comment (SagarKarwande) [inline apps/admin_app/lib/admin_app_web/templates/taxonomy/taxonomy.html.eex]: Wrong redirect page: Use `/taxonomy`

**Comment by SagarKarwande** [comment]
After new taxon addition there is no minus class applied on taxon.
This will need change in following template file:
`/snitch/apps/admin_app/lib/admin_app_web/templates/taxonomy/taxon.html.eex`

File: apps/admin_app/lib/admin_app_web/templates/payment_method/new.html.eex (hunk 1 above)
Comment (jaypal412) [reply apps/admin_app/lib/admin_app_web/templates/payment_method/new.html.eex]: Done

File: apps/admin_app/lib/admin_app_web/templates/option_type/index.html.eex (hunk 2 above)
Comment (jaypal412) [reply apps/admin_app/lib/admin_app_web/templates/option_type/index.html.eex]: Done

File: apps/admin_app/lib/admin_app_web/templates/taxonomy/index.html.eex (hunk 3 above)
Comment (jaypal412) [reply apps/admin_app/lib/admin_app_web/templates/taxonomy/index.html.eex]: Done

File: apps/admin_app/lib/admin_app_web/templates/taxonomy/taxonomy.html.eex (hunk 4 above)
Comment (jaypal412) [reply apps/admin_app/lib/admin_app_web/templates/taxonomy/taxonomy.html.eex]: Done

Comment (SagarKarwande) [reply apps/admin_app/lib/admin_app_web/templates/taxonomy/taxonomy.html.eex]: This change is remaining.

**Pull request merged**